### DISCUSSION TITLE
Prevent AI from spawning in certain area.

### DISCUSSION THREAD

**Discussion by ZaidRehan** [Vdauphin/HeartsAndMinds#1625] [Question and answer] [answered]
Hello, I'm trying to organize a starter FOB to start a H&M Campaign and want to know how to prevent AI from spawning in certain areas because enemy forces are spawning next to player spawn and attacking them.

**Comment by Vdauphin** [comment, marked as answer]
Hello,

You can do it the same as it is possible with base: http://vdauphin.github.io/HeartsAndMinds/Prevent-AI-from-spawning-near-the-base
Add marker with name and copy / paste the code with the correct marker name

Cheers

**Reply by ZaidRehan** [reply]
Thanks.
